### ISSUE TITLE
Update the charter or remove the link to it in README.md

### ISSUE THREAD

**Issue by xfq** [w3c/clreq#359]
In a [meeting](https://www.w3.org/2021/03/24-clreq-minutes.html#t04), Eric suggested that we either update our [charter](https://www.w3.org/International/clreq/charter/) or remove the link to it in `README.md`, because it is too old.

**Comment by r12a** [comment]
Yes, let's update it.  There is no formal process required.  The charter is simply intended to show current intentions, and can be updated at any time.

You could change the end date, or possibly even replace the date with No Fixed Date, or something like that.

The initial chair box reflects a very old situation, where the Chinese host was subsuming everything they wanted to be involved in under the clreq banner.  I suggest replacing the names with those of the current chairs.  The group could also consider whether to nominate some additional chairs.

Initial team contacts should be you and me.

Then the rest of the page should be rewritten to reflect current work and expected developments, relative to the currently constituted clreq group (ie. remove references to Mongolian, Tibetan, and Uighur).  The charter should be forward-looking, so there may be parts that should be trimmed because they are no longer relevant.

Basically, you should end up with a simple page that describes the work of the group and the intended achievements looking ahead.  This then will provide a document that describes an agreed common direction, purpose and process for the participants in the group.
